### ISSUE TITLE
Files: missing actions in folder context menus

### ISSUE THREAD

**Issue by lidel** [ipfs-shipyard/ipfs-webui#1631]
I noticed some actions are available via bottom drawer menu (when selecting a single folder), but are missing from the context menu (file list, breadcrumbs):

> ![2020-09-14--21-15-18](https://user-images.githubusercontent.com/157609/93128249-7c29d180-f6cf-11ea-9211-44ebdb30385a.png)

Any reason we should not show the same actions in both places? 

I did quick check and drawe menu actions work as expected: 

- :green_heart: Share link of a folder works as expected
- :green_heart: Download of a folder works as expected with fix from #1630 
- :green_heart: Delete of a folder works as expected
- :green_heart: / :broken_heart: Rename of a folder works fine, unless the directory is named `test` :upside_down_face:  (or is it just me?)

**Comment by rafaelramalho19** [comment]
Maybe you already had a directory called `test`?

**Comment by jessicaschilling** [comment]
AFAIK absolutely no reason not to put those options in right-click context menu ... if they work.

**Comment by jessicaschilling** [comment]
Added labels to this, moved it into "Quick Fixes" and the v2.12 milestone -- let's discuss if this doesn't seem viable.

**Comment by lidel** [comment]
cc @Swedneck – do you have similar issue with rename?

**Comment by rafaelramalho19** [comment]
@jessicaschilling I would change this to v2.11.2, since this is a important patch before next desktop release

**Comment by jessicaschilling** [comment]
Sounds good.

**Comment by Sovxx** [comment]
#1405 looks pretty similar

**Comment by ldeffenb** [comment]
@jessicaschilling I see the "bottom drawer menu" in the illustration above, but how do you get that in the Files page of the webui?  When I click on anything in the list, it drills into it if a directory or displays it if a file.  What's the magic to just "select" it to see the bottom drawer menu?

**Comment by jessicaschilling** [comment]
@ldeffenb The checkboxes to the left of each item in the list enable you to select multiple (or even just one) items and act on them:

![image](https://user-images.githubusercontent.com/1507828/94305414-5c47a880-ff2e-11ea-8114-eb73ac91c2af.png)

**Comment by ldeffenb** [comment]
@jessicaschilling Wow, NOW I see the checkbox.  My monitor's contrast and brightness were causing all of the lightly colored objects to all but disappear.  I could barely make out the "Browse" button and knew there was an entry field to the left of it.  But there was absolutely no trace of the checkbox.  I opened the same page on another machine and boom, the box(es) were there.  Then I played with the monitor contrast setting so I can now actually SEE the subtle items.  And strangely, I had to reduce the contrast from 100 to 50 to bring them to visibility.  A Samsung LS24E370DL HDMI monitor, not a cheap knockoff.

All is good now.  Thanks and I'm looking forward to the next update that fixes the menus I'm used to working with.
